samples/react-chat/src/App.tsx

```diff
@@ -1,10 +1,10 @@
-import React, { memo, useCallback, useState } from 'react';
+import { memo, useCallback, useState } from 'react';
 import './App.css';
 import { Login } from '@microsoft/mgt-react';
 import { Chat, ChatList, NewChat, ChatListButtonItem, ChatListMenuItem } from '@microsoft/mgt-chat';
 import { ChatMessage, Chat as GraphChat } from '@microsoft/microsoft-graph-types';
 import { Compose24Filled, Compose24Regular, bundleIcon } from '@fluentui/react-icons';
-import { GraphChatThread } from '../../../packages/mgt-chat/src/statefulClient/StatefulGraphChatListClient';
+import { GraphChatThread } from '@microsoft/mgt-chat/src/statefulClient/StatefulGraphChatListClient';
```

Comment:
why change to `@microsoft`?

Reply:
Follow the convention of how other mgt-chat components are being imported.

Reply:
I think the convention if we are following is to import from ```@microsoft/mgt-chat```